portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,17 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function showPicture() {
+    var image = document.getElementById("portrait");
```

Comment:
use `const` instead of `var`
and consider making `"portrait"` a const value that you can reference from outside.

Reply:
When I try making `"portrait"` a global const variable, I get: "**Uncaught TypeError: Cannot read property 'style' of null"**

I looked at StackOverflow and someone said its prolly because it executes before the DOM fully loads and he gave a solution, but I was wondering if there was an easier way to fix this

Reply:
discussed offline. not necessary to make portait a const